### DISCUSSION TITLE
Replacement of build & branch link graphs by the new dependency graph

### DISCUSSION THREAD

**Discussion by dcoraboeuf** [nemerosa/ontrack#1077] [Ideas]
As of version 4.5, the build & branch link graphs are considered deprecated and replaced by the more general (and faster) dependency graph (accessible from the build & branch pages, under _Tools > Dependency graph_).

Unless decided otherwise, the build & branch link graphs will be removed in version 4.6.

Please participate into this discussion if you have arguments for keeping these legacy graphs.

**Comment by rathpc** [comment]
I like having the full branch link graph to see the full picture of all dependencies

**Reply by dcoraboeuf** [reply]
Hi @rathpc , can you post an example (obfuscated of course) of the view you like?

**Reply by rathpc** [reply]
this one
![image](https://github.com/user-attachments/assets/29bc0268-617a-4663-b696-0b91e5766e00)

**Reply by dcoraboeuf** [reply]
Thanks :)

So that's the one at build level.

Does not the Next UI graph at build level provide the same thing?

In the Next UI, navigate to a build and click on "Links"? Your feedback would be very precious in allowing me to improve the Next UI pages.

**Reply by rathpc** [reply]
Ahh yes that view is perfect so no problem at all. The only recommendation I have is to make the borders on cards a bit darker as the very light gray is hard to distinguish when zoomed out a lot. Maybe `#777` and the arrow lines maybe `#999`?

**Reply by dcoraboeuf** [reply]
Great advice :) Thanks

**Reply by dcoraboeuf** [reply]
Created #1346 for follow-up.
